docs(loaders): Revamp Overview and Appearance pages (#863)
telerik/blazor-docs: components/loader/overview.md

This article provides information about the <a href = "https://www.telerik.com/blazor-ui/loader" target="_blank">Blazor Loader component</a> and its core features.

> The Loader component displays an animated loading indicator, which shows users that the app is working on something and they should wait. The component provides a variety of predefined animated graphics, colors and sizes.

Author: ```suggestion
The Loader component displays an animated loading indicator, which shows users that the app is working on something in the background. The component provides a variety of predefined animated graphics, colors and sizes.
```
```suggestion
The Loader component displays an animated loading indicator, which shows users that the app is working on something and they should wait. The component provides a variety of predefined animated graphics, colors and sizes.
```

> The Loader component displays an animated loading indicator, which shows users that the app is working on something and they should wait. The component provides a variety of predefined animated graphics, colors and sizes.

Author: ```suggestion
The Loader component displays an animated loading indicator, which shows users that the app is working on something in the background. The component provides a variety of predefined animated graphics, colors and sizes.
```
```suggestion
The Loader component displays an animated loading indicator, which shows users that the app is working on something and they should wait. The component provides a variety of predefined animated graphics, colors and sizes.
```

## Loader Parameters

The following table lists the Loader parameters. Also check the [Loader API Reference](/blazor-ui/api/Telerik.Blazor.Components.TelerikLoader).

<style>
    article style + table {
        table-layout: auto;
        word-break: normal;
    }
</style>

| Parameter | Type and Default&nbsp;Value | Description |
| --- | --- | --- |
| `Class` | `string` | Renders a custom CSS class to the `<div class="k-loader">` element. Use it to [override theme styles]({%slug themes-override%}). |
| `Size` | `string`<br />(`"md"`) | Sets the [size of the animated graphic]({%slug loader-appearance%}#size). For convenience, use the members of the static class [`ThemeConstants.Loader.Size`](/blazor-ui/api/Telerik.Blazor.ThemeConstants.Loader.Size). |
| `ThemeColor` | `string`<br />(`"primary"`) | Sets the [color of the animated graphic]({%slug loader-appearance%}#themecolor). For convenience, use the members of the static class [`ThemeConstants.Loader.ThemeColor`](/blazor-ui/api/Telerik.Blazor.ThemeConstants.Loader.ThemeColor). |
| `Type`| `LoaderType` enum<br />(`Pulsing`) | Defines the [loading animation shape]({%slug loader-appearance%}#type). |

> | `Visible` | `bool` | Sets if the Loader is rendered on the page. |

Author: ```suggestion
| `Visible` | `bool`<br /> (`true`) | Sets if the Loader is rendered on the page. |
```
